code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -176,7 +176,7 @@
 
 /obj/structure/closet/secure_closet/brig
 	name = "brig locker"
-	req_access = list(ACCESS_BRIG)
```

Comment:
Why not security access here?

Reply:
I believe this is the locker that officers use to secure prisoner's items, so it belongs to ACCESS_BRIG .

Reply:
But you’re also giving detective the access?

Reply:
Oh I think I see what you mean, I added forensics here when I don't believe I should've. I'll remove it.